WalletWasabi.Fluent/ViewModels/Wallets/Settings/WalletCoinJoinSettingsViewModel.cs

```diff
@@ -115,7 +115,8 @@ public void ManuallyUpdateOutputWalletList()
 		UiContext.WalletRepository.Wallets
 			.Connect()
 			.AutoRefresh(x => x.IsLoaded)
-			.Filter(x => (x.Id == _wallet.Id || x.Settings.OutputWalletId != _wallet.Id) && x.IsLoaded)
+			.Filter(x => (x.Id == _wallet.Id || x.Settings.OutputWalletId != _wallet.Id) &&
```

Comment:
I think this one line of change should fix this issue, bit I didn't test it:
```c#
.Filter(x => x.Id == _wallet.Id || (x.Settings.OutputWalletId != _wallet.Id && x.IsLoaded))
```

Reply:
With this change problem will still be reproducible in another scenario:
Open Wallet A, Create wallet B, Go to Wallet A's CJ setting and see than Wallet B is missing

Reply:
Because you are hiding the problem with your change but not touching the underlying cause which is that a wallet can be opened but not in `IsLoaded` State, in case `SkipSynchronization = true`